scripts/run_tails.py

```diff
@@ -145,17 +157,13 @@ def process_ccd_quad(
             x_o, y_o = predictions[ii, 1:] * stacks.shape[1]
             # save png
             plot_stack(
-                # tessellation[ii]['stack'],
                 tessellation[ii]["stack_raw"],
                 reticles=((x_o, y_o),),
                 w=6,
                 h=2,
                 dpi=360,
                 save=str(p_date / f"{name}_{ni}.png"),
-                # cmap=cmr.arctic,
-                # cmap=plt.cm.viridis,
-                # cmap=plt.cm.cividis,
-                cmap=plt.cm.bone,
+                cmap="bone",
```

Comment:
I have not used bone, but it seems good. Any particular reasons for choosing it?

Reply:
Experimented with a ton of other (default) options over the years and like this one the most. @stefanv's inventions like `viridis` are great, but I prefer a more monochrome feel for single-band optical images. BTW, we use `bone` on Fritz by default.